docs/sources/flow/when-to-use-flow/opentelemetry-collector.md

```diff
@@ -0,0 +1,109 @@
+
+The Collector may be a better choice if:
+* You are already using the Collector, and do not need any of the features of Flow.
+* You prefer YAML syntax over River.
+
+## Similarities
+
+### Many Collector components are available in Flow
+
+Flow contains many of the components available inside the Collector. 
+For example, Collector's OTLP exporter is available in Flow as `otelcol.exporter.otlp`.
+
+The Grafana Agent development team is actively working on adding more of the Collector's components to Flow.
+If a Flow user needs a Collector feature to be made available in the Agent, the Agent development team could work on implementing it.
+
+### Similar performance when processing OpenTelemetry signals natively
+
+Most of Flow's `otelcol` components are just thin wrappers over a Collector component.
+As a result, the CPU and memory performance of Flow is similar.
+
+## Differences
+
+### Configuration language
+
+Collector uses YAML for its configuration, and Flow uses River for its configuration.
+
+#### Example - coalesce
+
+One of River's main advantages is its standard library. It contains handy functions such as coalesce.
+You can use this function to retrieve the first argument which is not null or empty:
+
+```river
+```
+
+### Modularity
+
+The Agent configuration is flexible and modular:
+* Telemetry and non-telemetry components can be combined easily. 
+  For example, a configuration argument for `otelcol.exporter.otlp` could be supplied from the response of an HTTP request via `remote.http`.
+* Some or all of the components which an Agent runs could be defined outside of its main config file.
+  For example, whole snippets of Flow configuration could be loaded from a file (`module.file`), 
+  a Git repository (`module.git`), or a HTTP endpoint (`module.http`).
+
+#### Example - retrieving data from a file
+
+Let's say you want to use OAuth2 authentication in the Collector. If you need to retrieve `client_id` 
+or `client_secret` from a file, then you must use the `client_id_file` or `client_secret_file` config parameters.
+
+```yaml
+```
+
+In the Agent, you'd use `otelcol.auth.oauth2` with the normal `client_id` and `client_secret` parameters, 
+and you would set up another component that retrieves those from a `local.file` component. 
+
+```river
+```
+
+The string could also come from a `remote.s3`, `remote.vault`, or `remote.http`. 
+This gives Flow users lots of flexibility because strings coming out of those components 
+could be used for any parameter in any component which requires a string - not just for 
+a `client_id` for OAuth2.
+
+```river
+```
+
+### Flow can process Prometheus signals natively
+
+Collector needs to convert Prometheus signals to the OTLP format to process them.
+Flow, on the other hand, can process those signals natively using components such as `prometheus.relabel`, `prometheus.relabel`, and `prometheus.remote_write`.
+This could lead to better performance and ease of use.
```

Comment:
Our tests have shown that this leads to considerable performance gains.

@sh0rez , is there anything public we can link to?